app/controllers/v1/submission_files_controller.rb

```diff
@@ -13,7 +13,7 @@ def create
 
   def update
     submission_file = SubmissionFile.find(params[:id])
-    submission_file.rows = submission_file_params[:rows]
+    submission_file.rows = submission_file_params[:rows] if submission_file_params[:rows]
```

Comment:
How about using a guard clause? If there are no rows to update, we have nothing to do.